.github/ISSUE_TEMPLATE/test-failure.md

```diff
@@ -18,6 +18,10 @@ to reproduce locally
 
 **Applicable branches**:
 
+**Failure history**:
+Link to build stats and possible indication of when this started failing and how often it fails
```

Comment:
I think this should be in a comment block like the "fill out" instructions above.

Reply:
addressed in ac71c20